core/services/headtracker/head_tracker.go

```diff
@@ -374,6 +374,12 @@ func (ht *HeadTracker) Healthy() error {
 	return nil
```

Comment:
Since you access the saver through the `.Saver()` method perhaps some of the top-level wrapper methods on headtracker can be removed?

Reply:
Removed two odd methods. If I find anything else while processing head_tracker next time, there will be more cleanup.